### PR TITLE
Make the example more clear

### PR DESCRIPTION
**Pull request by Amir-61**
* Make the example more clear

/to: @davidcheung or @loay or @bajtos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ In LoopBack applications, add the following entry to your
   "final:after": {
     "strong-error-handler": {
       "params": {
+         "debug": true,
```
Comment (bajtos) [inline README.md]: By default, loopback-workspace scaffolds LoopBack applications with `debug: false` for security reasons. I think we should use `debug: false` here too, and add `server/middleware.development.json` with `debug:false`.

I see that we are using `debug:true`on L44 where we are showing usage in express apps. I think that place should be changed to `debug: app.get('env') === 'development'` or similar (I am not sure if my code snippet here works).

**Comment by Amir-61** [comment]
@bajtos Thanks for the great inputs...PTAL again :-)

File: README.md [hunk 2]
```diff
@@ -131,7 +133,7 @@ app.use(errorHandler({ log: false }));
   "final:after": {
     "strong-error-handler": {
       "params": {
-        "debug": true,
+        "debug": false,
```
Comment (bajtos) [inline README.md]: This should stay `true` - you want to see detailed error logs when running in development mode.

File: README.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ var app = express();
 // setup your routes
 // `options` are set to default values. For more info, see `options` below.
 // app.use(errorHandler({ /* options, see below */ }));
-app.use(errorHandler({debug:true, log:true}));
+app.use(errorHandler({debug: app.get('env') === 'development', log:true}));
```
Comment (bajtos) [inline README.md]: I think it would be better to split this onto multiple lines, put `debug` and `log` on their own lines.

**Comment by Amir-61** [comment]
@bajtos PTAL again. Thanks! 🙏

**Pull request merged**